### PR TITLE
Factory and Percy for Dear Internet Page

### LINKED ISSUE #5679: Factory and Percy - add DearInternetPage

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #5679

### CODE REVIEW

File: network-api/networkapi/utility/faker/streamfield_provider.py [hunk 1]
```diff
@@ -212,6 +212,43 @@ def generate_full_width_image_field():
...
+            f' is {" ".join(fake.words(nb=15))}. {fake.sentence()}',
+            '</p>',
+        )
+    )
+
+    letter = f'<p>{fake.paragraph(nb_sentences=10, variable_nb_sentences=True)}</p>'
+
+    attributes = {
+        'author_description': author_description,
+        'author': author,
```
Comment (Pomax) [inline network-api/networkapi/utility/faker/streamfield_provider.py:236]: should this be `author_name`? `author` no longer exists

File: network-api/networkapi/wagtailpages/factory/dear_internet_page.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    letters_section_heading = Faker('sentence')
+    letters = Faker('streamfield', fields=['letter']*15)
+    cta = Faker('paragraph', nb_sentences=6, variable_nb_sentences=True)
+    cta_button_text = 'Donate'
+    cta_button_link = 'https://donate.mozilla.org'
+
+
+def generate(seed):
+    home_page = get_homepage()
+    reseed(seed)
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/factory/dear_internet_page.py:36]:
```suggestion

    reseed(seed)
```

**Review by Pomax — CHANGES_REQUESTED** [review]

File: network-api/networkapi/utility/faker/streamfield_provider.py (hunk 1 above)
Comment (mmmavis) [reply network-api/networkapi/utility/faker/streamfield_provider.py]: you are totally right! I replaced text on the wrong line 😂

**Comment by Pomax** [comment]
See the following in travis:
image

**Review by Pomax — CHANGES_REQUESTED** [review]
errors on travis

**Comment by mmmavis** [comment]
hopefully I didn't leave any silly mistakes unfixed 😂

**Review by Pomax — APPROVED** [review]

**Comment by Pomax** [comment]
:shipit:

**Pull request merged**